crates/astria-sequencer/src/bridge/state_ext.rs

```diff
@@ -1216,56 +1050,61 @@ mod test {
 
         // write to second
         let rollup_id_1 = RollupId::new([2u8; 32]);
-        deposit = Deposit {
+        let deposit_1 = Deposit {
             rollup_id: rollup_id_1,
             source_action_index: 1,
-            ..deposit
+            ..deposit.clone()
         };
         state
-            .put_deposit_event(deposit.clone())
+            .put_deposit_event(deposit_1.clone())
             .await
             .expect("writing deposit events for rollup 2 should be ok");
 
-        // delete all info
+        // delete nonce info
         state
-            .clear_block_deposits()
+            .clear_block_deposit_nonces()
             .await
             .expect("clearing deposits call should not fail");
+        // check that nonces were deleted
         assert_eq!(
-            state
-                .get_deposit_events(&rollup_id)
+            get_deposit_nonce(&state, &rollup_id)
                 .await
                 .expect("deposit should return empty when none exists"),
-            vec![],
-            "deposits were cleared and should return empty vector"
+            0u32,
+            "nonce should have been deleted also"
         );
-        // check that all info was deleted
         assert_eq!(
-            state
-                .get_deposit_events(&rollup_id_1)
+            get_deposit_nonce(&state, &rollup_id_1)
                 .await
                 .expect("deposit should return empty when none exists"),
-            vec![],
-            "deposits were cleared and should return empty vector"
+            0u32,
+            "nonce should have been deleted also"
         );
+        // check that deposits were not deleted
         assert_eq!(
-            state
-                .get_deposit_nonce(&rollup_id)
+            super::get_deposit_events(&state, &rollup_id)
                 .await
                 .expect("deposit should return empty when none exists"),
-            0u32,
-            "nonce should have been deleted also"
+            vec![deposit],
+            "deposits were cleared and should return empty vector"
```

Comment:
Is this accurate? I may be mistaken, but it seems like there specifically should be a deposit in the vector returned